plugins/xivo-yealink/v83/templates/common/dect_model.tpl

```diff
@@ -1,5 +1,6 @@
 #!version:1.0.0.1
 
+static.auto_provision.server.url = http://{{ ip }}:{{ http_port }}/
```

Comment:
Why not https?

Reply:
Because provd does not support HTTPS for provisioning currently. It has HTTPS only for the REST API.